docs/setup.md

```diff
@@ -29,11 +29,12 @@ On a high level, installing Draupnir works like the following;
 3. Configure draupnir see [further below](#post-install).
 4. Start draupnir.
 
-Draupnir can be installed in two ways, via Docker, or building it yourself.
+Draupnir can be installed in three ways, via Docker, building it yourself or via [matrix-docker-ansible-deploy](https://github.com/spantaleev/matrix-docker-ansible-deploy).
 
 See the below links for corresponding installation documentation;
 - [Docker](./setup_docker.md)
 - [Building It](./setup_selfbuild.md)
+- [matrix-docker-ansible-deploy](https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-bot-draupnir.md)
```

Comment:
Should this instead link to https://github.com/spantaleev/matrix-docker-ansible-deploy/blob/master/docs/configuring-playbook-bot-draupnir.md ?

Reply:
I think both links should link to this, or no link for mdad. I know it's a tricky one but people are more likely to click the first link